module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaAuditSignInLogs.md

```diff
@@ -34,66 +34,69 @@ Get-EntraBetaAuditSignInLogs
-                                                                                                                                                                ionProtocol
---                                   --------------                     -----                                ------------------------ ------------------------- -----------
-903c0263-3ddb-409c-a248-07edf1967200 Microsoft Graph Command Line Tools 14d82eec-204b-4c2f-b7e8-296a70dab67e                          {}                        none
+
+```Output
+Id                                   AppDisplayName                                                 AppId                                AppTokenProtectionStatus AuthenticationMethodsUsed AuthenticationProtocol
+--                                   --------------                                                 -----                                ------------------------ ------------------------- ----------------------
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb Azure Active Directory PowerShell  00001111-aaaa-2222-bbbb-3333cccc4444                                                  ropc                   si...
+bbbbbbbb-1111-2222-3333-cccccccccccc Azure Portal                       11112222-bbbb-3333-cccc-4444dddd5555                        {}                        none
 ```
-This example returns the first n logs.
 
-### Example 3: Get audit logs containing a given ActivityDisplayName
+This example returns the first two audit logs of sign-ins.
+
+### Example 3: Get audit logs containing a given AppDisplayName
 
 ```powershell
- Get-EntraBetaAuditSignInLogs -Filter "ActivityDisplayName eq 'Add owner to application'"
- Get-EntraBetaAuditSignInLogs -Filter "ActivityDisplayName eq 'Add owner to application'" -Top 1
+Connect-Entra -Scopes 'AuditLog.Read.All and Directory.Read.All'
+Get-EntraBetaAuditSignInLogs -Filter "AppDisplayName eq 'Graph Explorer'" 
+Get-EntraBetaAuditSignInLogs -Filter "AppDisplayName eq 'Graph Explorer'" -Top 1
 ```
-These commands show how to get sign-in logs by ActivityDisplayName.
-
-### Example 4: Get all sign-in logs with a given result
 
-```powershell
- Get-EntraBetaAuditSignInLogs -Filter "result eq 'success'"
- Get-EntraBetaAuditSignInLogs -Filter "result eq 'failure'" -Top 1
+```Output
+Id                                   AppDisplayName                                                 AppId                                AppTokenProtectionStatus AuthenticationMethodsUsed AuthenticationProtocol
+--                                   --------------                                                 -----                                ------------------------ ------------------------- ----------------------
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb Graph Explorer PowerShell  00001111-aaaa-2222-bbbb-3333cccc4444   
 ```
-These commands show how to get sign-in logs by the result.
+
+This example demonstrates how to retrieve sign-in logs by AppDisplayName.
 
```

Comment:
```suggestion

### Example 4: Get all sign-in logs between dates

```powershell
Connect-Entra -Scopes 'AuditLog.Read.All','Directory.Read.All'
Get-EntraBetaAuditSignInLogs -Filter "createdDateTime ge 2024-07-01T00:00:00Z and createdDateTime le 2024-07-14T23:59:59Z"
```

This example shows how to retrieve sign-in logs between dates.

```

Reply:
changes done